### ISSUE TITLE
'yuu' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by Zt-freak** [noaione/yuu#27]
I ran `pip install yuu`

this is what I got:
`Collecting yuu
  Downloading https://files.pythonhosted.org/packages/ce/29/6ed68468efa9cdcf884e7c0eed022247709cc584c8e22bfc24fc4b62bdf4/yuu-1.2.2-py2.py3-none-any.whl
Collecting tqdm (from yuu)
  Downloading https://files.pythonhosted.org/packages/1c/1a/cd6ee6b8b06557dcc5590785af2fe90fa10c19c28e567c1e3a299d5081e7/tqdm-4.44.1-py2.py3-none-any.whl (60kB)
     |████████████████████████████████| 61kB 230kB/s
Collecting m3u8 (from yuu)
  Downloading https://files.pythonhosted.org/packages/bd/46/0bb9b9ca48785611d1aea9df3f5a5d057de183251db464359286ba597b8f/m3u8-0.5.4-py3-none-any.whl
Collecting pycryptodome (from yuu)
  Downloading https://files.pythonhosted.org/packages/95/0d/db748eb8f68a56cc97e2d0bc62c26186999e7a074e166fc6186ce3449ebe/pycryptodome-3.9.7-cp37-cp37m-win_amd64.whl (14.1MB)
     |████████████████████████████████| 14.1MB 218kB/s
Collecting requests[socks] (from yuu)
  Downloading https://files.pythonhosted.org/packages/1a/70/1935c770cb3be6e3a8b78ced23d7e0f3b187f5cbfab4749523ed65d7c9b1/requests-2.23.0-py2.py3-none-any.whl (58kB)
     |████████████████████████████████| 61kB 393kB/s
Collecting click (from yuu)
  Downloading https://files.pythonhosted.org/packages/dd/c0/4d8f43a9b16e289f36478422031b8a63b54b6ac3b1ba605d602f10dd54d6/click-7.1.1-py2.py3-none-any.whl (82kB)
     |████████████████████████████████| 92kB 392kB/s
Collecting iso8601 (from m3u8->yuu)
  Downloading https://files.pythonhosted.org/packages/ef/57/7162609dab394d38bbc7077b7ba0a6f10fb09d8b7701ea56fa1edc0c4345/iso8601-0.1.12-py2.py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests[socks]->yuu)
  Downloading https://files.pythonhosted.org/packages/b9/63/df50cac98ea0d5b006c55a399c3bf1db9da7b5a24de7890bc9cfd5dd9e99/certifi-2019.11.28-py2.py3-none-any.whl (156kB)
     |████████████████████████████████| 163kB 297kB/s
Collecting idna<3,>=2.5 (from requests[socks]->yuu)
  Downloading https://files.pythonhosted.org/packages/89/e3/afebe61c546d18fb1709a61bee788254b40e736cff7271c7de5de2dc4128/idna-2.9-py2.py3-none-any.whl (58kB)
     |████████████████████████████████| 61kB 302kB/s
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 (from requests[socks]->yuu)
  Downloading https://files.pythonhosted.org/packages/e8/74/6e4f91745020f967d09332bb2b8b9b10090957334692eb88ea4afe91b77f/urllib3-1.25.8-py2.py3-none-any.whl (125kB)
     |████████████████████████████████| 133kB 285kB/s
Collecting chardet<4,>=3.0.2 (from requests[socks]->yuu)
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)
     |████████████████████████████████| 143kB 285kB/s
Collecting PySocks!=1.5.7,>=1.5.6; extra == "socks" (from requests[socks]->yuu)
  Downloading https://files.pythonhosted.org/packages/8d/59/b4572118e098ac8e46e399a1dd0f2d85403ce8bbaad9ec79373ed6badaf9/PySocks-1.7.1-py3-none-any.whl
Installing collected packages: tqdm, iso8601, m3u8, pycryptodome, certifi, idna, urllib3, chardet, PySocks, requests, click, yuu
  WARNING: The script tqdm.exe is installed in 'C:\Users\Bryan\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script chardetect.exe is installed in 'C:\Users\Bryan\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script yuu.exe is installed in 'C:\Users\Bryan\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed PySocks-1.7.1 certifi-2019.11.28 chardet-3.0.4 click-7.1.1 idna-2.9 iso8601-0.1.12 m3u8-0.5.4 pycryptodome-3.9.7 requests-2.23.0 tqdm-4.44.1 urllib3-1.25.8 yuu-1.2.2
WARNING: You are using pip version 19.2.3, however version 20.0.2 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.`

But if I run `yuu`, I get:
`'yuu' is not recognized as an internal or external command,
operable program or batch file.`

how do I run yuu?

**Comment by matchajam** [comment]
I have the same issues, have you fix yours?

**Comment by Zt-freak** [comment]
No, but it could be that you have to add it to your PATH Environment Variable.
